modules/c/pages/gs-downloads.adoc

```diff
@@ -1,60 +1,1207 @@
+
 = Download Links
-ifdef::show_edition[:page-edition: {release}]
-ifdef::prerelease[:page-status: {prerelease}]
 :page-role:
-:pag-toclevels: 3@
 :description: Download links for Couchbase Lite for C
 :keywords: edge nosql api
 
-// BEGIN -- DO NOT EDIT
-include::partial$_set_page_context_for_c.adoc[]
 
-:param-abstract!:
-include::{root-partials}_show_page_header_block.adoc[]
-// END -- DO NOT EDIT
+:source-language: c
+
+:source-language: c
 
-// tag::installation[]
-:param-page: {page-relative-src-path}
-include::{root-partials}_show_get_started_topic_group.adoc[]
 
+[abstract]
+--
+Description -- _{description}_ +
+--
+
+
+// tag::installation[]
+Steps in Getting Started::
+xref:c:gs-install.adoc[Install]
+ |
+xref:c:gs-build.adoc[Build and Run]
 
-:download-path: {url-downloads-mobile}
-:beta-url: https://packages.couchbase.com/releases/couchbase-lite-c/{version-full}/
 
- 
 .Downloads are available for the following versions:
 ****
-<<release-3-2-2>>   |
 <<release-3-2-1>>   |
 <<release-3-2-0>>   |
 <<vs-release-1-0-0>>   |
-// | 
+// |
 ****
 
 // This block will always represent the major release version
-:param-version: 3.2.2
-:param-version-hyphenated: 3-2-2
-include::partial$downloadslist.adoc[]
```

Comment:
I'd be tempted to retain the `downloadslist.adoc` partial myself, as the long lists full of download URLs are easy to make *mistakes in* if we try to edit them manually.

But leaving as is for @simon-dew to look and make a call!